lib/couchdb/index.js

```diff
@@ -84,23 +85,59 @@ const upsert = R.curry((couch, bucket, keys, doc) =>
 )(doc)))
 
 
+const _findOneByIdWhereEq = R.curry((couch, bucket, id, predicates) =>
+  getById(couch, bucket, id)
+  .then(R.when(
```

Comment:
Why the need for the `doc` variable?  I believe you could simplify this like the following:

```javascript

const _findOneByIdWhereEq = R.curry((couch, bucket, id, predicates) =>
  getById(couch, bucket, id)

  .then(R.unless( 
    R.whereEq( ID.couchify(predicates) )
  , R.always(null)
  ))  
```

I'm not 100% sure, but I believe this is the same.